lib/active_record/connection_adapters/sunstone/schema_statements.rb

```diff
@@ -19,17 +19,27 @@ def columns(table_name)
           end
         end
 
+        def definition(table_name)
```

Comment:
maybe rename function to table_definition,

Also did we want to move this to a global schema and not per table?